package.json

```diff
@@ -1,4 +1,6 @@
 {
+  "name": "@openfeature/specification",
```

Comment:
name insures that the package-lock.json name will not be different if someone uses a different directory name for their cloned project like I did. If this happens, the TOC check can fail.